charts/metallb/templates/deprecated_configInline.yaml

```diff
@@ -0,0 +1,3 @@
+{{- if .Values.configInline }}
```

Comment:
As we don't have a release yet maybe we can just remove this

Reply:
Hmm, there will be probably users that upgrade from the upstream chart (older-version) to ours?

Reply:
Good point.